docs/API.md

```diff
@@ -466,16 +477,13 @@ An object with the following properties:
 > TODO: Clarify the default value of each optional `"params"` property.
 >
 
-*priotity*
+priority parameter
 
-This parameter is a free integer that will be used by the zonemaster testing backend daemon to sort the test requests from highest to lowest priority.
-This parameter will typically be used in a setup where a GUI will send requests to the bachend and would like to get response as soon as possible while at the same time using the idle time for background bach testing.
-The drawback of this setup will be that the GUI will have to wait for at least one background processing slot to become free (would be a few secods in a typical installation with up to 30 parallel zonemaster processes allowed)
+see the *priority* section
 
-*queue*
+queue parameter
```

Comment:
Please update the `"queue"` item in the list of params above, instead of having a separate section here.